doc/develop/modules.rst

```diff
@@ -569,6 +569,36 @@ Build files located in a ``MODULE_EXT_ROOT`` can be described as:
 This allows control of the build inclusion to be described externally to the
 Zephyr module.
 
+.. _modules-vulnerability-monitoring:
+
+Vulnerability monitoring
+========================
+
+The module description file :file:`zephyr/module.yml` can be used to improve vulnerability monitoring.
+
+If your module needs to track vulnerabilities using an external reference
+(e.g your module is forked from an other repository), you can use the ``vulnerabilities`` section.
+It contains the field ``external-references`` that contains a list of references that needs to
+be monitored for your module. The supported format are:
+
+- CPE (Common Platform Enumeration)
+- PURL (Package URL)
+
+.. code-block:: yaml
```

Comment:
it might be my own preference but I would also add an actual example of how things would look like with real values, ex. by using mbed TLS, or whatever, as an example. Also, as a way to make sure people instantly realize where the new section belongs, i would have the YAML snippet show more than just the relevant section, ex. include name and build entries

```
name: lvgl
build:
  ...
vulnerabilities:
   external-references:
     - ...
```